recipes/emsdk/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "3.1.69":
+    url: "https://github.com/emscripten-core/emsdk/archive/3.1.60.tar.gz"
```

Comment:
Something I noticed while I was checking this version update is that the source version is updated, but this url is still pointing to version 3.1.60.

Reply:
@SonarShroom 
Thank you for your comment!
I fixed url and sha256.